test/Index/roles-contained.swift

```diff
@@ -13,16 +15,18 @@ struct Wrapped<T> {
 
 // Begin tests
 
+let _: Int = intValue
+// CHECK: [[@LINE-1]]:8 | struct/Swift | Int | {{.*}} | Ref | rel: 0
+// CHECK: [[@LINE-2]]:14 | variable/Swift | intValue | {{.*}} | Ref,Read | rel: 0
+
 let typedProperty: Int = 1
-// CHECK: [[@LINE-1]]:5 | variable/Swift | typedProperty | {{.*}} | Def | rel: 0
```

Comment:
I removed some of these `Def` checks to keep this file focused on `RelCont`. `Def` should already be suitably covered by other tests.